core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1858,8 +1857,14 @@ class KafkaApis(val requestChannel: RequestChannel,
                 if (resource.name.isEmpty)
                   throw new InvalidRequestException("Invalid empty resource name")
                 auth.addAcls(immutable.Set(acl), resource)
+                if (logger.isDebugEnabled) {
+                  logger.debug("added acl " + acl + " to " + resource)
```

Comment:
Could we improve the `Authorizer` logs so that this is redundant?

Nit: we normally start log messages with a capital letter. Also, string interpolation is preferred over concatenation. Same for the other debug statement.